src/util/dispatch-resize-event.ts

```diff
@@ -1,11 +1,4 @@
 export const dispatchResizeEvent = () => {
-    /*
-     * The shorthand (`window.dispatchEvent(new Event('resize'));`)
-     * causes compiler errors, so we go the long way around.
-     * See https://stackoverflow.com/a/1818513/280972
-     * /Ads
-     */
-    const resizeEvent = window.document.createEvent('UIEvents');
-    resizeEvent.initUIEvent('resize', true, false, window, 0);
+    const resizeEvent = new UIEvent('resize', { view: window, detail: 0 });
```

Comment:
😞

This isn't working as it should in Firefox on Windows:

![image](https://user-images.githubusercontent.com/154725/65141876-cf603a00-da11-11e9-82dc-aa9bd3e95028.png)

(Don't mind the colour, that's just to make sure the browser has actually loaded from the branch, and isn't using a cached copy.)